tests/testthat/test-rerun.R

```diff
@@ -1,17 +1,28 @@
+test_that("is deprecated", {
+  expect_snapshot({
+    . <- rerun(1, 1)
+  })
+
+})
+
 test_that("single unnamed arg doesn't get extra list", {
+  options(lifecycle_verbosity = "quiet")
```

Comment:
Here and elsewhere: Should be `local_options()`. Could also be pulled at top level since the whole file tests a deprecated function.

Reply:
Doh